src/main/java/com/redhat/devtools/intellij/knative/Constants.java

```diff
@@ -12,4 +12,5 @@
 
 public class Constants {
     public static final String STRUCTURE_PROPERTY = Constants.class.getPackage().getName() + ".structure";
+    public static final String TOOLBAR_PLACE = "Knative.View";
```

Comment:
better to add the package name here as the structure so if there will ever be another plugin that uses this name in IJ we are safe.